src/components/AllArticles.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import {useEffect, useState} from "react";
+import {getArticles} from "../utils.js";
+import NewsArticles from './NewsArticles.jsx';
+import { Flex } from "antd";
+
+
+function AllArticles() {
+
+    const [newsData, setNewsData] = useState([]);
+    const [loading, setLoading] = useState(true);
+
+    // const [filter, setFilter] = useState("All Items");
```

Comment:
As a rule you should remove commented out code by the time you're submitting a PR, to keep it clean!